src/bot/admin.py

```diff
@@ -23,7 +26,9 @@ def __init__(self, bot):
     @commands.has_permissions(manage_server=True)
     async def reloadall(self, ctx):
         """Reload all currently loaded Cogs."""
-        # Cast to list to prevent a RuntimeError, since we just want the iteration for the names of the Cogs
+
+        # Cast to list to prevent a RuntimeError
+        # since we just want the iteration for the names of the Cogs
```

Comment:
I understand you need to make a copy in case the collection changes size, but you don't need to mutate it. Use a `tuple()`.